applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -319,6 +319,17 @@ build_bridge(#state{endpoints=Endpoints
             ])
           ,wh_json:get_value(<<"Custom-Channel-Vars">>, JObj, wh_json:new())
          ),
+
+    EndpointFilter = fun(Element) ->
+        case Element of
+            {<<"Outbound-Caller-ID-Number">>, Number} -> false;
+            {<<"Outbound-Caller-ID-Name">>, Name}     -> false;
+            _OtherValues                              -> true
+        end
+    end,
+
+    FmtEndpoints = format_endpoints(Endpoints, Number, JObj, AccountId, EndpointFilter),
+
```

Comment:
When trying to move this to a separate function I realized why I used a fun here also, because we need to match against Name and Number, which are in the scope of that function.  Using a fun means that we do not have to pass these parameters all the way through to the filter function.

Given this I no longer agree with moving the fun to a function in this specific case.